backend/dataall/cdkproxy/stacks/policies/mlstudio.py

```diff
@@ -1,7 +1,7 @@
 from dataall.cdkproxy.stacks.policies.service_policy import ServicePolicy
 
 from dataall.db import permissions
-from dataall.modules.common.sagemaker.cdk.statements import create_sagemaker_statements
+from dataall.modules.sagemaker_base.cdk.statements import create_sagemaker_statements
 
```

Comment:
I think we will discuss this today, but I am more inclined to separate notebooks and ML Studio permissions to follow the least-privilege principle. Even if that means to have some duplicated permissions

Reply:
Yes, I also think that a module just for one file is too much :) 
I believe we can merge this and rework it in ML studio or in other PR. But I will add a todo to delete it